W-18708117-Add-info-about-AI-doc-in-Exchange-mk (#472)
mulesoft/docs-code-builder: modules/ROOT/pages/des-publish-api-spec-to-exchange.adoc

image::des-api-exchange.png["API specification in Exchange"]

> You can select the API, navigate through its summary, and see all the endpoints you defined in the previous tasks. You can also review AI-generated doc or make changes right in the asset details page in Exchange. For more information, see https://docs.mulesoft.com/exchange/generating-documentation-with-ai[Generating Documentation with AI^].

Reviewer: ```suggestion
You can select the API, navigate through its summary, and see all the endpoints you defined in the previous tasks. You can also generate documentation for REST APIs without documentation on the asset details page in Exchange. For more information, see https://docs.mulesoft.com/exchange/generating-documentation-with-ai[Generating API Documentation with Einstein Generative AI^].
```

Reviewer: For this release, only REST APIs are supported. We will include more APIs later this year.
